src/Php82/bootstrap.php

```diff
@@ -30,3 +30,7 @@ function odbc_connection_string_should_quote(string $str): bool { return p\Php82
 if (!function_exists('odbc_connection_string_quote')) {
     function odbc_connection_string_quote(string $str): string { return p\Php82::odbc_connection_string_quote($str); }
 }
+
+if (!function_exists('ini_parse_quantity')) {
+    function ini_parse_quantity(string $shorthand): string { return p\Php82::ini_parse_quantity($shorthand); }
```

Comment:
the return type looks suspicious to me here

Reply:
Dang.  Copy/paste error.

Reply:
Sorry for that.  I'm rushing as I'm likely to be unavailable for most of the next week.  Fixed.